src/main/java/Task.java

```diff
@@ -0,0 +1,149 @@
+//import java.util.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Task {
```

Comment:
Clear code👍, but I feel that the class is too long. Consider dividing the task class into different types :)

Reply:
alright! will work on it, thanks!